<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitConfigDialog</class>
 <widget class="QDialog" name="GitConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="GitConfigWidget" name="cfgSystem">
      <attribute name="title">
       <string>System</string>
      </attribute>
     </widget>
     <widget class="GitConfigWidget" name="cfgUser">
      <attribute name="title">
       <string>User</string>
      </attribute>
     </widget>
     <widget class="GitConfigWidget" name="cfgRepository">
      <attribute name="title">
       <string>Repository</string>
      </attribute>
     </widget>
     <widget class="GitConfigWidget" name="cfgCombined">
      <attribute name="title">
       <string>Effective Configuration</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GitConfigWidget</class>
   <extends>QWidget</extends>
   <header>GitConfigWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GitConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GitConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
